sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensorTest.java

```diff
@@ -124,7 +124,7 @@ public void execute_warn_about_deprecated_integration_tests() throws IOException
     new CoverageReportImportSensor(coverageConf, coverageAggregator, "cs", "C#", true)
       .execute(context);
 
-    assertThat(logTester.logs(LoggerLevel.WARN)).containsOnly("Starting with SonarQube 6.2 separation between Unit Tests and Integration Tests "
+    assertThat(logTester.logs(LoggerLevel.WARN)).contains("Starting with SonarQube 6.2 separation between Unit Tests and Integration Tests "
```

Comment:
What other messages are now being produced as a result of this commit?

Reply:
>"Failed to set working directory hidden: C:\Users\Andrei\AppData\Local\Temp\junit4479429460697924433\junit8811176124595669393\.sonar"

This is happening inside [AbstractProjectOrModule](https://github.com/SonarSource/sonarqube/blob/master/sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/AbstractProjectOrModule.java#L101) when it tries to
```
        Files.createDirectories(workingDir);
        Files.setAttribute(workingDir, "dos:hidden", true, LinkOption.NOFOLLOW_LINKS);
```

Reply:
@dbmeneses could you please help to investigate this `Failed to set working directory hidden`. Is it fixed in more recent versions of the sonar-plugin-api?

Reply:
It's done in the scanner-engine. At one point it was failing sometimes when the directory didn't exist and it was fixed. @andrei-epure-sonarsource if possible, could you run it with verbose to get a stack trace?

Reply:
@dbmeneses it's because I was using SQ API 7.6 instead of 7.7. Updating to 7.7 removed the extra warnings